templates/components/news-vertical-teaser/news-vertical-teaser.twig

```diff
@@ -16,14 +16,13 @@
 {%
   set classes = [
   'su-card',
-  'su-news-vertical-teaser',
-  modifier_class
+  'su-news-vertical-teaser'
 ]
 %}
 {% set attributes = attributes.addClass(classes) %}
 
-<article {{ attributes }}>
-  <figure class="su-media su-media--image su-card__media">
+<article {{ attributes }} class="{{ modifier_class }}">
+  <figure class="su-media  su-media--image su-card__media">
```

Comment:
This will produce duplicate `class` attributes on the article wrapper. This is no bueno. Also, where did code climate go. There are a number of formatting issues in this PR. Don't retroactively fix anything. I'll merge it into the wip on the default-content branch. 

@boznik